satellite/repair/repair_test.go

```diff
@@ -51,8 +51,6 @@ func TestDataRepair(t *testing.T) {
 		// first, upload some remote data
 		uplinkPeer := planet.Uplinks[0]
 		satellite := planet.Satellites[0]
-		// stop discovery service so that we do not get a race condition when we delete nodes from overlay
-		satellite.Discovery.Service.Refresh.Stop()
```

Comment:
Unsure if we need to stop each storagenode.contact.chore.loop in these tests. The tests pass without however

Reply:
If a node could be added back to the overlay after we delete it, we might want to worry about it, even if it is unlikely. If the storagenode contact chore loop interval is decreased significantly (like to 1-10ms or something), will the test still pass? If so, I would say don't worry about it. Otherwise, we should probably stop those loops.

Reply:
good point, I'll test it with a short interval

Reply:
All the tests even when making the interval 1ms so i think we're ok